devops/test_configs.json

```diff
@@ -35,11 +35,11 @@
       "cmake_args": ""
     },
     {
-      "config": "ocl_gen9",
-      "name": "OCL GEN9 LLVM Test Suite",
+      "config": "ocl_gen12",
+      "name": "OCL GEN12 LLVM Test Suite",
```

Comment:
Do we still use this? Can we remove this part of the config?

Reply:
Same as above.